Changelog.md

```diff
@@ -4,6 +4,8 @@ Features:
 
 Bugfixes:
 
+ * Code generator: enable access to events of base contracts' names
```

Comment:
This is both a modification of the code generator and the name resolution stage. Perhaps we can just make it a feature?